docs/contributing/release-process.md

```diff
@@ -73,7 +73,7 @@ git add . && git commit -m "Release ${VERSION}"
 This is the "release commit". Push and create a PR.
 
 ```
-gh pr create -f -l tide/merge-method-squash
+gh pr create -f -l tide/merge-method-squash -B release-1.25
```

Comment:
As a separate PR we should update the master-branch version of this document to specify -B, mentioning the value to use for a branch.

Reply:
Yeah, was a miss. Removed.